Release 1.13.0 (#816)
astronomer/astronomer-providers: CHANGELOG.rst

Changelog
=========

1.13.0 (2022-12-16)
-------------------

Feature
"""""""

> - Support assume role for AWS base async hook

Reviewer: Please write this from a usage perspective

Author: Fixed 
- Fix ``AwsBaseHookAsync`` to support assume role ``role_arn`` passed via ``extra_config`` in the airflow connection.

Reviewer: It is an enhancement rather than a fix.

Enhance ``AwsBaseHookAsync`` to support assume role role_arn passed via extra_config in the airflow connection.

  (`#804 <https://github.com/astronomer/astronomer-providers/pull/804>`_)
- Support private key connection in ``SFTPHookAsync``
  (`#749 <https://github.com/astronomer/astronomer-providers/pull/749>`_)

Bug Fixes
"""""""""

> - Consider kubernetes connection details in trigger and hook async.

Reviewer: Please write this from a usage perspective

Author: Fixed to 
- Fix ``KubernetesPodOperatorAsync`` to consider kubernetes connection id and details in trigger.

Reviewer: what is `details in trigger.` ?

Author: Fixed to 
- Fix ``KubernetesPodOperatorAsync`` to consider kubernetes connection id in the trigger.

> - Fix ``KubernetesPodOperatorAsync`` to consider kubernetes connection id and details in trigger.

Reviewer 2: ```suggestion
- Fix ``KubernetesPodOperatorAsync`` to consider kubernetes connection id in the trigger.
```

Author: Fixed to

Fix KubernetesPodOperatorAsync to consider kubernetes connection id in the trigger.
